app/src/ui/check-runs/ci-check-run-popover.tsx

```diff
@@ -218,6 +218,10 @@ export class CICheckRunPopover extends React.PureComponent<
 
   private renderRerunButton = () => {
     const { checkRuns } = this.state
+    if (this.props.repository.isGHE) {
```

Comment:
While it'd be nice to have a more reusable approach here I'm proposing we stick with our typical way of selectively targeting GHES here and potentially consider a more generalized approach separately.

```suggestion
    // GHES doesn't support rerunning checks yet
    if (this.props.repository.endpoint !== getDotComAPIEndpoint()) {
      return null
    }
```